cms/models.py

```diff
@@ -79,6 +139,8 @@ class Meta:
         StreamFieldPanel("content"),
     ]
 
+    subpage_types = [WhoShouldEnrollPage]
```

Comment:
is it not expecting a string in it ? Actually, I am passing string. 
`subpage_types = ['WhoShouldEnrollPage']`

Reply:
You could pass in a string to lazy-load, that also works but we already have access to the class so it isn't really required to pass in a string, we can give it the actual model.